### ISSUE TITLE
ERRO[0004] Failed to start transient timer unit: Unit f6cfd6ea082535d93ea8063060bb5261d73f1cf66fb22d1b6403b9070b7b001c.timer already exists. f6cfd6ea082535d93ea8063060bb5261d73f1cf66fb22d1b6403b9070b7b001c

### ISSUE THREAD

**Issue by atmecs123** [containers/podman#15583]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind bug

**Description**

**Describe the results you received:**
[root@ci-5250vm029 ~]# docker restart webui
ERRO[0004] Failed to start transient timer unit: Unit f6cfd6ea082535d93ea8063060bb5261d73f1cf66fb22d1b6403b9070b7b001c.timer already exists.
f6cfd6ea082535d93ea8063060bb5261d73f1cf66fb22d1b6403b9070b7b001c

**Describe the results you expected:**
Podman container must be restart successfully.

**Output of `podman version`:**

```
[root@ci-5250vm029 ~]# podman version
Client:       Podman Engine
Version:      4.1.1
API Version:  4.1.1
Go Version:   go1.17.7
Built:        Mon Jul 11 14:56:53 2022
OS/Arch:      linux/amd64
[root@ci-5250vm029 ~]#
```

**Output of `podman info`:**

```
host:
  arch: amd64
  buildahVersion: 1.26.2
  cgroupControllers:
  - cpuset
  - cpu
  - cpuacct
  - blkio
  - memory
  - devices
  - freezer
  - net_cls
  - perf_event
  - net_prio
  - hugetlb
  - pids
  - rdma
  cgroupManager: systemd
  cgroupVersion: v1
  conmon:
    package: conmon-2.1.2-2.module+el8.6.0+15917+093ca6f8.x86_64
    path: /usr/bin/conmon
    version: 'conmon version 2.1.2, commit: 8c4f33ac0dcf558874b453d5027028b18d1502db'
  cpuUtilization:
    idlePercent: 91.94
    systemPercent: 2.09
    userPercent: 5.98
  cpus: 4
  distribution:
    distribution: '"rhel"'
    version: "8.6"
  eventLogger: file
  hostname: ci-5250vm029
  idMappings:
    gidmap: null
    uidmap: null
  kernel: 4.18.0-372.19.1.el8_6.x86_64
  linkmode: dynamic
  logDriver: k8s-file
  memFree: 10001514496
  memTotal: 33440468992
  networkBackend: cni
  ociRuntime:
    name: runc
    package: runc-1.0.3-2.module+el8.6.0+14877+f643d2d6.x86_64
    path: /usr/bin/runc
    version: |-
      runc version 1.0.3
      spec: 1.0.2-dev
      go: go1.17.7
      libseccomp: 2.5.2
  os: linux
  remoteSocket:
    exists: true
    path: /run/podman/podman.sock
  security:
    apparmorEnabled: false
    capabilities: CAP_AUDIT_WRITE,CAP_CHOWN,CAP_DAC_OVERRIDE,CAP_FOWNER,CAP_FSETID,CAP_KILL,CAP_MKNOD,CAP_NET_BIND_SERVICE,CAP_NET_RAW,CAP_SETFCAP,CAP_SETGID,CAP_SETPCAP,CAP_SETUID,CAP_SYS_CHROOT
    rootless: false
    seccompEnabled: true
    seccompProfilePath: /etc/opt/veritas/flex/seccomp.json
    selinuxEnabled: true
  serviceIsRemote: false
  slirp4netns:
    executable: /bin/slirp4netns
    package: slirp4netns-1.2.0-2.module+el8.6.0+15917+093ca6f8.x86_64
    version: |-
      slirp4netns version 1.2.0
      commit: 656041d45cfca7a4176f6b7eed9e4fe6c11e8383
      libslirp: 4.4.0
      SLIRP_CONFIG_VERSION_MAX: 3
      libseccomp: 2.5.2
  swapFree: 17169498112
  swapTotal: 17179865088
  uptime: 21h 4m 27.2s (Approximately 0.88 days)
plugins:
  log:
  - k8s-file
  - none
  - passthrough
  - journald
  network:
  - bridge
  - macvlan
  - ipvlan
  volume:
  - local
  - filevol
  - veritas
registries:
  docker.io:
    Blocked: true
    Insecure: false
    Location: docker.io
    MirrorByDigestOnly: false
    Mirrors: null
    Prefix: docker.io
    PullFromMirror: ""
  flex.io:
    Blocked: false
    Insecure: false
    Location: console:8443
    MirrorByDigestOnly: false
    Mirrors: null
    Prefix: flex.io
    PullFromMirror: ""
  registry.access.redhat.com:
    Blocked: true
    Insecure: false
    Location: registry.access.redhat.com
    MirrorByDigestOnly: false
    Mirrors: null
    Prefix: registry.access.redhat.com
    PullFromMirror: ""
  registry.redhat.io:
    Blocked: true
    Insecure: false
    Location: registry.redhat.io
    MirrorByDigestOnly: false
    Mirrors: null
    Prefix: registry.redhat.io
    PullFromMirror: ""
  search:
  - flex.io
store:
  configFile: /etc/containers/storage.conf
  containerStore:
    number: 9
    paused: 0
    running: 9
    stopped: 0
  graphDriverName: overlay
  graphOptions:
    overlay.mountopt: nodev,metacopy=on
  graphRoot: /var/lib/containers/storage
  graphRootAllocated: 73364480000
  graphRootUsed: 5161345024
  graphStatus:
    Backing Filesystem: xfs
    Native Overlay Diff: "false"
    Supports d_type: "true"
    Using metacopy: "true"
  imageCopyTmpDir: /tmp
  imageStore:
    number: 9
  runRoot: /run/containers/storage
  volumePath: /var/lib/containers/storage/volumes
version:
  APIVersion: 4.1.1
  Built: 1657551413
  BuiltTime: Mon Jul 11 14:56:53 2022
  GitCommit: ""
  GoVersion: go1.17.7
  Os: linux
  OsArch: linux/amd64
  Version: 4.1.1

```

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

```
[root@ci-5250vm029 ~]# rpm -q podman
podman-4.1.1-2.module+el8.6.0+15917+093ca6f8.x86_64
[root@ci-5250vm029 ~]#

```

**Additional environment details (AWS, VirtualBox, physical, etc.):**
```
[root@ci-5250vm029 ~]# uname -r
4.18.0-372.19.1.el8_6.x86_64
[root@ci-5250vm029 ~]# cat /etc/os-release
NAME="Red Hat Enterprise Linux"
VERSION="8.6 (Ootpa)"
ID="rhel"
ID_LIKE="fedora"
VERSION_ID="8.6"
PLATFORM_ID="platform:el8"
PRETTY_NAME="Red Hat Enterprise Linux 8.6 (Ootpa)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:redhat:enterprise_linux:8::baseos"
HOME_URL="https://www.redhat.com/"
DOCUMENTATION_URL="https://access.redhat.com/documentation/red_hat_enterprise_linux/8/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"

REDHAT_BUGZILLA_PRODUCT="Red Hat Enterprise Linux 8"
REDHAT_BUGZILLA_PRODUCT_VERSION=8.6
REDHAT_SUPPORT_PRODUCT="Red Hat Enterprise Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="8.6"
[root@ci-5250vm029 ~]#
```

**runc package version:**
```
[root@ci-5250vm029 ~]# rpm -qa | grep runc
runc-1.0.3-2.module+el8.6.0+14877+f643d2d6.x86_64
[root@ci-5250vm029 ~]#

```

**Comment by vrothberg** [comment]
Thanks for reaching out, @atmecs123.

Can you share how the `webui` container has been created?  Since the report indicates that you're running on RHEL.  If desire a potential backport, please reach out through the Red Hat customer channels (e.g., Bugzilla).

**Comment by atmecs123** [comment]
As per this(https://access.redhat.com/solutions/6961531) the issue was seen and was resolved in runc version 1.0.3. Still we are seeing this issue.
webui container was created using below command,
/bin/podman run -d -it ${options} --read-only --name ${cname} \
                  --network podman ${extra_hosts} \
                  -p ${VIPescaped}:443:443 -p ${VIPescaped}:8443:8443 -p ${VIPescaped}:1443:1443 \
                  -v ${cert_dir}:${cert_dir}:z,ro \
                  -v ${webui_certstore_path}:${webui_certstore_path}:Z,ro \
                  -v ${eca_certstore_path}:${eca_certstore_path}:Z \
                  -v ${smartcard_ca_path}:${smartcard_ca_path}:Z \
                  -v ${usercerts_path}:${usercerts_path}:Z \
                  -v ${i18n_common_path}:${i18n_common_path}:ro \
                  -v ${i18n_gateway_path}:${i18n_gateway_path}:ro \
                  -v ${webui_conf_path}:/etc/nginx/nginx.conf:z,ro \
                  -v ${nginx_conf_path}:/etc/nginx/conf.d/infra.conf:z,ro \
                  -v ${nginx_sub_conf_path}:/etc/nginx/conf.d/flex:Z \
                  -v ${upgrade_commit_flag_path}:${upgrade_commit_flag_path}:Z \
                  -v ${software_path}:/mnt/software:z \
                  -v ${webui_tmp_path}:/var/lib/nginx/tmp:Z \
                  --tmpfs /tmp:rw,nodev,nosuid,noexec,size=32m,mode=1777 \
                  -e "ENV_WEBCONSOLE_NAME=${web_console}" \
                  ${image_tag}

**Comment by vrothberg** [comment]
Thank you! In that case, please reach out via the Red Hat customer channels.  There is not much we can do upstream.
